tgui/packages/tgui/interfaces/NifPanel.js

```diff
@@ -0,0 +1,204 @@
+import { useBackend, useLocalState } from '../backend';
+import { Window } from '../layouts';
+import { Box, Dropdown, LabeledList, ProgressBar, Section, Button, Input, BlockQuote, Flex, Collapsible, Table } from '../components';
+import { TableCell, TableRow } from '../components/Table';
+
+export const NifPanel = (props, context) => {
+  const { act, data } = useBackend(context);
+  const { linked_mob_name, loaded_nifsofts, max_nifsofts, max_power, current_theme } = data;
+  const [settingsOpen, setSettingsOpen] = useLocalState(
+    context,
+    settingsOpen,
+    false
+  );
+
+  return (
+    <Window
+      title={'Nanite Implant Framework'}
+      width={500}
+      height={400}
+      resizable
+      theme={current_theme}>
+      <Window.Content>
+        <Section
+          title={'Welcome to your NIF, ' + linked_mob_name}
+          buttons={
+            <Button
+              icon="cogs"
+              tooltip="NIF Settings"
+              tooltiptooltipPosition="bottom-end"
+              selected={settingsOpen}
+              onClick={() => setSettingsOpen(!settingsOpen)}
+            />
+          }>
+          {(settingsOpen && <NifSettings />) || <NifStats />}
+          {(!settingsOpen &&
+            <Section title={'NIFSoft Programs (' + (max_nifsofts - loaded_nifsofts.length) + " Slots Remaining)"} right>
```

Comment:
```suggestion
            <Section title={`NIFSoft Programs (${max_nifsofts - loaded_nifsofts.length} Slots Remaining)`} right>
```
Similar to above, but template literals really shine here.